packages/ui/src/composites/wui-list-content/styles.ts

```diff
@@ -0,0 +1,24 @@
+import { css } from 'lit'
+
+export default css`
+  :host {
+    display: flex;
+    column-gap: var(--wui-spacing-s);
+    padding: 17px 18px 17px var(--wui-spacing-m);
+    width: 100%;
+    background-color: var(--wui-gray-glass-002);
+    border-radius: var(--wui-border-radius-xs);
+    color: var(--wui-color-fg-250);
+  }
+
+  wui-image {
+    width: 20px;
+    height: 20px;
+    border-radius: var(--wui-border-radius-3xl);
+  }
+
+  wui-icon {
+    width: 20px;
```

Comment:
```suggestion
    width: var(--wui-icon-size-lg);
```

What about using variables?